opencti-platform/opencti-front/lang/front/en.json

```diff
@@ -3228,6 +3232,7 @@
   "Subscriptions & digests": "Subscriptions & digests",
   "Subsectors": "Subsectors",
   "Success": "Success",
+  "Success the enrollment of your OCTI is done": "Success the enrollment of your OCTI is done",
```

Comment:
Not sure on the sentence here....
"Your OpenCTI platform is successfully enrolled" ?

Please do not use abbreviation such as `OCTI` on UI text